src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -45,27 +45,14 @@ class ErrorHandlerMiddleware implements MiddlewareInterface
     /**
      * Default configuration values.
      *
-     * Ignored if contructor is passed an ErrorHandler instance.
+     * Ignored if contructor is passed an ExceptionTrap instance.
      *
-     * - `log` Enable logging of exceptions.
-     * - `skipLog` List of exceptions to skip logging. Exceptions that
-     *   extend one of the listed exceptions will also not be logged. Example:
-     *
-     *   ```
-     *   'skipLog' => ['Cake\Error\NotFoundException', 'Cake\Error\UnauthorizedException']
-     *   ```
-     *
-     * - `trace` Should error logs include stack traces?
-     * - `exceptionRenderer` The renderer instance or class name to use or a callable factory
-     *   which returns a \Cake\Error\ExceptionRendererInterface instance.
-     *   Defaults to \Cake\Error\ExceptionRenderer
+     * Configuration keys and values are shared with `ExceptionTrap`.
      *
      * @var array<string, mixed>
+     * @see \Cake\Error\ExceptionTrap
      */
     protected $_defaultConfig = [
-        'skipLog' => [],
```

Comment:
Should we document configs that are pass-through? That's always been an unsure thing for me.

Reply:
Added something to the `_defaultConfig` docs. Let me know if you think there should be more content there.

Reply:
@ADmad What do you think?

Reply:
This works I guess.

Reply:
pass-through options are never going to look completely clean

Reply:
As I see there are 2 options. Point to class where the options are eventually used, as done currently. Else duplicate the option descriptions at both places.